scripts/lochness_create_template.py

```diff
@@ -368,31 +364,31 @@ def create_config_template(config_loc: Path, args: object) -> None:
         for study in args.studies:
             line_to_add = f'''
     {study}:
-        base: /example_box_root/{study}
+        base: ProNET/{study}
         delete_on_success: False
         file_patterns:
             actigraphy:
-                - vendor: Philips
-                  product: Actiwatch 2
-                  data_dir: actigraphy
-                  pattern: '*csv'
-                  protect: True
-                - vendor: Activinsights
-                  product: GENEActiv
-                  data_dir: actigraphy
-                  pattern: '*csv'
-                - vendor: Insights
-                  product: GENEActivQC
-                  data_dir: actigraphy
-                  pattern: '*csv'
+                   - vendor: Activinsights
+                     product: GENEActiv
+                     data_dir: {study}_Actigraphy
+                     pattern: '*.*'
             eeg:
                    - product: eeg
-                     data_dir: eeg
-                     pattern: '*.csv'
+                     data_dir: {study}_EEG
+                     pattern: '*.*'
             interviews:
-                   - product: offsite_interview
-                     data_dir: interviews
-                     pattern: '*.mp4'
+                   - product: open
+                     data_dir: {study}_Interviews/OPEN
+                     out_dir: open
+                     pattern: '*.*'
+                   - product: psychs
+                     data_dir: {study}_Interviews/PSYCHS
+                     out_dir: psychs
+                     pattern: '*.*'
+                   - product: transcripts
+                     data_dir: {study}_Interviews/transcripts/Approved
```

Comment:
Did we include the `{study}_{datatype}` naming conventions in your SOP?

Reply:
Yes. Pronet team have added this and I've raised this to DPACC a few weeks ago. Pronet team is going through official agreements with other parties to make this finalized in the SOP, but yes, this will be the case.